tests/test_all_fields.py

```diff
@@ -474,6 +465,9 @@ def all_fields_test(self, streams_to_test):
                                 expected_field_value = expected_record.get(field, "EXPECTED IS MISSING FIELD")
                                 actual_field_value = actual_record.get(field, "ACTUAL IS MISSING FIELD")
 
+                                # to fix the failure warning of `created` for `invoices` stream
+                                if stream == 'invoices' and expected_field_value != "EXPECTED IS MISSING FIELD" and field == 'created':
+                                    expected_field_value = int(self.dt_to_ts(expected_field_value))
```

Comment:
I don't think this is a valid transformation. The created should be transformed using the pattern in base. Although I don't think changing it to a timestamp is a valid transformation, shouldn't this be in the singer datetime format?

Reply:
The discrepancy in this created field is that the `created` field is not in the singer time format in the schema, rather in the integer format. We did not change the data-type to avoid column split because the created field was already there with an integer type, and handled that in this test case.

Reply:
I don't think this change should not have been made. The bug tracker that was removed describes the fix that is needed. If the column splitting is a concern it is worth raising that to product or Krishnan. But this issue was not addressed yet so the Bug marker should not be removed removed. For your reference https://jira.talendforge.org/browse/TDL-13711.

Reply:
Yes, got your point. So do you suggest that we should continue showing the warning message for the `created` field, or we should add the bug tracker here?

Reply:
Either works so long as it can be easily found and used to reproduce the issue in that ticket.

Reply:
Sure, we have added the bugtracker in the code.